### DISCUSSION TITLE
add Georgian ( ka ) language to spaCy

### DISCUSSION THREAD

**Discussion by ghost** [explosion/spaCy#13135] [Language Support]
Hello, I would like to add support for Georgian ( ka ) language to SpaCy. currently we lack the support not from SpaCy but any other open source NLP libraries. SpaCy is the best NLP library in my opinion so, I wish to add the support here. 
I would like to know what do you think about it, and if it sounds applicable to you, I'll invest all my effort to get it ASAP.  
I would really appreciate, the link to the documentation about how to add completely new language 🙏  💯

**Comment by adrianeboyd** [comment]
It'd be great to add Georgian!

To add "alpha" support for a new language, you can see the types of language data/info required here: https://spacy.io/usage/linguistic-features#language-data

The basics are:

* `__init__.py` with the language class and defaults class
* `punctuation.py` + `tokenizer_exceptions.py` for tokenization
* `examples.py` with a few example sentences
* basic tests, usually including a few tokenization examples

And frequently the initial support also includes:

* `lex_attrs.py` for `LIKE_NUM` and similar

Here's a good example of a recent PR that adds two new languages: https://github.com/explosion/spaCy/pull/13116

**Reply by ghost** [reply]
huge thanks! 🥇

**Comment by Gvantsakhubulia** [comment]
Hello. 
Did you add it?
